client/src/components/Footer.js

```diff
@@ -57,9 +60,10 @@ class Footer extends React.Component {
           </div>
           <div className="row text-center">
             <div className="col-md-12 col-lg-12">
-              <p onClick={this.redirectToOther} className="copyright">
-                © 2019 Copyright: Penn State Computational Biomechanics Group
-              </p>
+            <p>
+                    Contact Us: info@NSFCAREER.IO <br />
+                    <span onClick={()=>window.open('')}>  IP </span> | Privacy Policy &amp; <span onClick={()=>window.open('')}>  IRB </span> | Collaborate
```

Comment:
I said you need to create new pages, not just open new windows. Make pages for these.

Reply:
I apologize for this. but I don't have the design of the page. could you please provide.